01_mpileups/gzip/gzip.cwl

```diff
@@ -1,12 +1,16 @@
 cwlVersion: v1.2
 class: CommandLineTool
-baseCommand: gzip
+
+hints:
+    DockerRequirement:
+      dockerPull: alpine@sha256:06b5d462c92fc39303e6363c65e074559f8d6b1363250027ed5053557e3398c5
```

Comment:
Any chance we can use a tagged image (for example `alpine:20220715`).

Reply:
If you want. The image associated with a tag can change, which can be undesirable. In this case, it's fine, but it's something to remember when working with docker images. The image associated with a particular sha hash won't change.

Reply:
I guess I trust the tag to represent some intended stationary state of the image, which I guess it might change, but that may indeed be desirable (e.g. to fix a security vulnerability in the base image). Generally, I find the tag useful to know which version/release represents